scripts/build/Platform/Windows/upload_metrics_windows.cmd

```diff
@@ -0,0 +1,14 @@
+@ECHO OFF
+REM
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM
+REM
+
+ECHO [ci_build] aws s3 cp --recursive %CSV_DIRECTORY% %BUCKET% --acl bucket-owner-full-control
```

Comment:
@o3de/sig-build is [ci_build] an appropriate tag to log here?

Reply:
This is probably not something we need to emit, as the command below will be visible as well. Is this possibly a debug message?

Reply:
Removed debug line